examples/component/sqldb-postgres-query/go.mod

```diff
@@ -0,0 +1,34 @@
+module sqldb-postgres-query
+
+go 1.23.0
+
+toolchain go1.23.2
+
+require (
+	github.com/stretchr/testify v1.9.0
+	go.bytecodealliance.org v0.5.0
+	go.bytecodealliance.org/cm v0.1.0
```

Comment:
I would prefer it if we stuck to [the version `go.wasmcloud.dev/component` specifies](https://github.com/wasmCloud/go/blob/main/component/go.mod#L7) and update all of the examples once we update the component